mdtraj/core/topology.py

```diff
@@ -1169,6 +1177,8 @@ class Residue(object):
         The chain within which this residue belongs
     resSeq : int
         The residue sequence number
+    segment_id : str
+        An optional label for the segment to which this residue belongs
     """
 
     def __init__(self, name, index, chain, resSeq):
```

Comment:
Thanks Robert

Okay, let's change the constructor. I can do that and add tests. The writer
doesn't support this feature get. An additional point is I have an
improvement to pdstructure which fixes some edge cases related to this and
makes the ATOM parser match the letter of the law of the newer pdb spec at
the 70-80 character range. I didn't know if I should add this in the same
commit?
On Thu, Dec 3, 2015 at 16:59 Robert T. McGibbon <notifications@github.com>
wrote:

> In mdtraj/core/topology.py
> <https://github.com/mdtraj/mdtraj/pull/1001#discussion_r46626340>:
>
> > @@ -1169,6 +1177,8 @@ class Residue(object):
> >          The chain within which this residue belongs
> >      resSeq : int
> >          The residue sequence number
> > +    segment_id : str
> > +        An optional label for the segment to which this residue belongs
> >      """
> >
> >      def __init__(self, name, index, chain, resSeq):
>
> It will also need to be added to the constructor here, probably with a
> default value of the empty string.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mdtraj/mdtraj/pull/1001/files#r46626340>.
>